pkg/reaper/secrets.go

```diff
@@ -21,6 +25,16 @@ var EnableCassAuthVar = &corev1.EnvVar{
 	Value: "true",
 }
 
+var EnableAuthVar = &corev1.EnvVar{
```

Comment:
I think this is unused now?

Reply:
It is used [here](https://github.com/adejanovski/k8ssandra-operator/blob/enable-reaper-ui-auth/controllers/reaper/reaper_controller.go#L348).

Reply:
Right, sorry.